src/app/modules/item/pages/item-details/item-details.component.ts

```diff
@@ -23,8 +26,9 @@ export class ItemDetailsComponent implements OnDestroy {
   taskTabs: TaskTab[] = [];
   taskView?: TaskTab['view'];
 
-  fullFrameContent$ = this.layoutService.fullFrameContent$;
+  readonly fullFrameContent$ = this.layoutService.fullFrameContent$;
   readonly watchedGroup$ = this.userService.watchedGroup$;
+  readonly content$ = this.currentContentService.content$.pipe(filter(isItemInfo));
```

Comment:
But that's the same here. Items push info into the current content, components on the right side (excluding the top bar) of the screen are part of the content so should not use current content.